packages/components/src/components/stepper-horizontal/stepper-horizontal/stepper-horizontal.tsx

```diff
@@ -102,14 +91,17 @@ export class StepperHorizontal {
     }
   }
 
+  public componentDidRender(): void {
```

Comment:
We should do this in `componentDidLoad` like in every other component and not do it on every rerender.  
Are you sure you tried that lifecycle and not `componentWillLoad` by accident?